sections/sprint-1/pozyx.tex

```diff
@@ -33,6 +33,34 @@ \subsection{Using UWB}
 High bandwidth means faster data transfer, which most people would prefer, but if everyone were to use the same frequency the signals would interfere with each other, therefore the use of high-frequency signals is tightly regulated.
 Pozyx can use 500 MHz because it transmits at very low power.
 
+\subsection{Alternatives to UWB}
+Having a high accuracy is essential for the users' experience since a low accuracy could result in the in-game information being incorrect or just too imprecise to make it a joyful experience.
+While Pozyx uses UWB for positioning, it is interesting to take a look at the primary alternatives to indoor positioning and to see if any of them can compete with Pozyx' high accuracy \ref{pozyx}.
+
+\subsubsection{GPS}
+The obvious alternative for positioning data is GPS, which is frequently used for outdoors positioning.
+In optimal outdoor situations, GPS can provide accuracy within 4.9 meters \ref{gpsgov:accuracy}.
+However, the GPS positioning accuracy can be degraded due to buildings, bridges, and trees, making it a sub-optimal choice for indoor positioning.
```

Comment:
Evt smid en kommentar omkring at selvom præcisionen var 4.9 meter, så vil det stadigvæk være alt for upræcist for os.